sdlib/d/gc/page.d

```diff
@@ -939,13 +939,14 @@ private:
 				auto npages = e.npages;
 				scope(success) i += npages;
 
-				auto w = e.gcWord.load();
 				auto ec = pd.extentClass;
+				auto markBits = e.scanningMarkBitsSparse(gcCycle);
+
 				if (ec.isLarge()) {
```

Comment:
You are checking for large here, so that does not make sense to branch in scanningMarkBitsSparse as well, IMO, you can just have 2 methods.

Reply:
fixed